Homework3/task7 additionally.py

```diff
@@ -0,0 +1,32 @@
+# 7. Шоколадка **
+# Шоколадка имеет вид прямоугольника, разделенного на n×m долек.
+# Шоколадку можно один раз разломить по прямой на две части.
+# 1.Определите, можно ли таким образом отломить от шоколадки ровно k долек.
+
+n = int(input('n= ', ))
+m = int(input('m= ', ))
+k = int(input('k= ', ))
+if n * m >= k and (k % m == 0 or k % n == 0):
+    print('1.YES')
+else:
+    print('1.NO')
+
+# 2.Определите, можно ли отломить от шоколадки ровно k одиночных долек.
+if (k == 2 and (m, n == 1, 2 or n, m == 1, 2)) or\
+        (k == 1 and (m == 1 or n == 1)):
+    print('2.YES')
+else:
+    print('2.NO')
+# Т. к. ломаем только один раз по одной линиии, могут быть только 2 случая,
```

Comment:
we can cut any times we need

Reply:
but then the number of required slices should just be less than NxM

Reply:
and "k" must be a multiple of "N" or "M"